lib/cli/clients/icgc/icgc_script.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+import logging
+import os
+
+
+def icgc_call(objectid, token, toolpath, output):
+    logger = logging.getLogger('__log__')
+
+    os.environ['ACCESSTOKEN'] = token
+    call_args = [toolpath, 'download', '--object-id', objectid, '--output-dir', output]
+    logger.debug(call_args)
+    call = subprocess.Popen(call_args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    output, _ = call.communicate()
```

Comment:
Not a fan of `_` as a variable name. Something descriptive like `error` or `err_msg` would be better.